### PR TITLE
Initial spec of what we pin on PC

### PR DESCRIPTION
**Pull request by ThomasArts**

### CODE REVIEW

**Review by happi — APPROVED** [review]
How do you verify that the "claimer" on the child chain is the "poster" on the parent chain?

File: Periodically-Syncing-HyperChains.md [hunk 1]
```diff
@@ -225,7 +225,19 @@ sequenceDiagram
 
 ```
 
+### Pinning format
+
+The transaction posted to the PC is a hash of the following data drived from a specific block on the CC:
```
Comment (dincho) [inline Periodically-Syncing-HyperChains.md]: Do you mean derived?

Comment (ThomasArts) [inline Periodically-Syncing-HyperChains.md]:
```suggestion
The transaction posted to the PC is a hash of the following data derived from a specific block on the CC:
```

**Comment by ThomasArts** [comment]
> How do you verify that the "claimer" on the child chain is the "poster" on the parent chain?

we do not need to verify this. Anyone can post, but there is only one stakeholder that gets rewarded for it, so most incentive for that one to ask someone with a parent chain key to do the posting.

**Pull request merged**